modules/arm_plugin/thirdparty/CMakeLists.txt

```diff
@@ -86,7 +86,12 @@ else()
         if(ARM)
             list(APPEND ARM_COMPUTE_OPTIONS arch=armv7a)
         elseif(AARCH64)
-            list(APPEND ARM_COMPUTE_OPTIONS arch=arm64-v8a)
+            if(APPLE)
+                # Apple M1 is assumed
+                list(APPEND ARM_COMPUTE_OPTIONS arch=arm64-v8.2-a)
```

Comment:
What's the reason to choose different target architecture for M1?

Reply:
because it supports FP16 kernels.